src/languages/dns.js

```diff
@@ -0,0 +1,33 @@
+/*
+Language: DNS Zone file
+Author: Tim Schumacher <tim@datenknoten.me>
+Category: common, config
+*/
+
+function(hljs) {
+  return {
+    aliases: ['bind','zone'],
+    keywords: {
+      keyword:
+        'IN A AAAA AFSDB APL CAA CDNSKEY CDS CERT CNAME DHCID DLV DNAME DNSKEY DS HIP IPSECKEY KEY KX LOC MX NAPTR NS NSEC NSEC3 NSEC3PARAM PTR RRSIG RP SIG SOA SRV SSHFP TA TKEY TLSA TSIG TXT'
+    },
+    contains: [
+      hljs.COMMENT(';', '$'),
+	  {
+		className: 'operator', 
+		begin: '\\$(TTL|GENERATE|INCLUDE|ORIGIN)',
+		relevance: 10
```

Comment:
Is this so necessary? Maybe we can just add `keywords: 'TTL GENERATE INCLUDE ORIGIN'` to this mode?